ir_cdk_stacks/in_clw_01_stack.py

```diff
@@ -0,0 +1,127 @@
+                    "aws.logs"
+                ]
+            }
+
+            ep2 = {
+                "source": [
+                    "aws.cloudwatch"
+                ]
+            }
+
+            rule1 = events.Rule(self,
+                               "cdkRule1_clw",
+                               description='Rule created by CLW CDK',
+                               enabled=True,
+                               rule_name="rule1bycdk_clw",
+                               event_pattern=ep1)
+
+            rule2 = events.Rule(self,
+                                "cdkRule2_clw",
+                                description='Rule created by CLW CDK',
+                                enabled=True,
+                                rule_name="rule2bycdk_clw",
+                                event_pattern=ep2)
+
+            # 3. Create response lambda and add it as a target of the rule
+            lambda_dir_path = os.path.join(os.getcwd(), "ir_cdk_stacks", "in_clw_01")
+            response_lambda = _lambda.Function(
+                self,
+                "InClw01ResponseFunction",
+                runtime=_lambda.Runtime.PYTHON_3_7,
+                handler="clwUnauthAccessResponse.lambda_handler",
+                code=_lambda.Code.from_asset(lambda_dir_path),
+                function_name="InClw01ResponseFunction",
+                environment = {
+                    "webhook_url": SLACK_WEBHOOK_URL,
+                    "white_list_group": WHITE_LIST_GROUP,
+                }
+            )
+
+            response_lambda.add_to_role_policy(
+                iam.PolicyStatement(
+                    actions=["*"],
+                    effect=iam.Effect.ALLOW, resources=["*"],
+                )
+            )
+
+
+            rule1.add_target(event_target.LambdaFunction(response_lambda))
+            rule2.add_target(event_target.LambdaFunction(response_lambda))
+
+
+            # 4. Create SNS topic and subscription
+            topic = sns.Topic(self, "CDKCLWAccess", topic_name="CDKCLWAccess")
+
+            topic.add_subscription(subs.EmailSubscription(NOTIFY_EMAIL))
+            #topic.add_subscription(subs.EmailSubscription('k.singh.1@unsw.edu.au'))
+
+
+
+            # 5. Create IAM allow/deny policy
+            clwDenyAccessPolicy1 = iam.ManagedPolicy(self,
+                                             "InCLW01DenyPolicy1",
+                                             managed_policy_name = "ClWDenyAccess1",
+                                             statements=[
+                                                 iam.PolicyStatement(
+                                                     effect=iam.Effect.DENY,
+                                                     actions=["logs:*"],
+                                                     resources=["*"]
+                                                 )
+                                             ])
+
+            clwDenyAccessPolicy2 = iam.ManagedPolicy(self,
```

Comment:
Inject these policies into the lambda

Reply:
stack is creating the IAM policies which are used in lambda once created

Reply:
yeah but you're not injecting the name of the policy into the lambda

Reply:
I commented out the static url. using the names of policies in response lambda